Example/iOS Example/AppDelegate.swift

```diff
@@ -43,6 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return
         }
         PiwikTracker.configureSharedInstance(withSiteID: siteid, baseURL: baseurl)
+        PiwikTracker.shared?.logger?.minLevel = .info
```

Comment:
Maybe create an issue for tracking this refactoring if we think it is a good idea.